core/src/main/java/com/opentable/logging/LogMetadata.java

```diff
@@ -34,7 +35,7 @@
     private final List<Object> inlines = new ArrayList<>();
 
     private LogMetadata(Map<String, Object> metadata) {
-        this.metadata = metadata;
+        this.metadata = new LinkedHashMap<>(metadata); // in case we mutate it later
```

Comment:
It's to allow you to attach arbitrary fields to a logging event without having to extend it wholesale.  It's linked to preserve ordering in case that's useful to you.  (infrastructure doesn't care, but it's relatively cheap)